roachtest: alterpk-tpcc failed

[(roachtest).alterpk-tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1790595&tab=buildLog) on [master@954fe69d554162aec0fbc001aad1fe5103d8df13](https://github.com/cockroachdb/cockroach/commits/954fe69d554162aec0fbc001aad1fe5103d8df13):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200306-1790595/alterpk-tpcc/run_1
	test_runner.go:756: test timed out (10h0m0s)

```

<details><summary>More</summary><p>


Artifacts: [/alterpk-tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1790595&tab=artifacts#/alterpk-tpcc)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aalterpk-tpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).alterpk-tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1816964&tab=buildLog) on [master@d571624ae4f833c49c717728a74cc7be78a791f0](https://github.com/cockroachdb/cockroach/commits/d571624ae4f833c49c717728a74cc7be78a791f0):

```
	cluster.go:1935,alterpk.go:163,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2010
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: output in %s
		    -- arg 1: <string>
		  - output in run_081031.299_n4_workload_check_tpcc:
		  - error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:406
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2019
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2000
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1816964-1584603715-12-n4cpu4:4 -- ./workload check tpcc --warehouses 500 --expensive-checks {pgurl:1} returned:
		    stderr:
		    I200319 08:10:32.914531 1 workload/tpcc/tpcc.go:372  check 3.3.2.1 took 56.33867ms
		    I200319 08:10:46.288969 1 workload/tpcc/tpcc.go:372  check 3.3.2.2 took 13.374378691s
		    I200319 08:10:47.721635 1 workload/tpcc/tpcc.go:372  check 3.3.2.3 took 1.432598089s
		    I200319 08:11:32.886678 1 workload/tpcc/tpcc.go:372  check 3.3.2.4 took 45.164991168s
		    I200319 08:14:03.467908 1 workload/tpcc/tpcc.go:372  check 3.3.2.5 took 2m30.580935102s
		    I200319 08:17:03.106152 1 workload/tpcc/tpcc.go:372  check 3.3.2.6 took 2m59.637891081s
		    Error: check failed: 3.3.2.6: pq: communication error: rpc error: code = Canceled desc = context canceled
		    Error:  exit status 1
		    
		    stdout::
		  - exit status 1

```

<details><summary>More</summary><p>


Artifacts: [/alterpk-tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1816964&tab=artifacts#/alterpk-tpcc)
Related:
- #46136 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003112118_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003112118_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aalterpk-tpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).alterpk-tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1819517&tab=buildLog) on [master@bcffc157e3af316c2bfcf4850e67205010425211](https://github.com/cockroachdb/cockroach/commits/bcffc157e3af316c2bfcf4850e67205010425211):

```
	cluster.go:1935,alterpk.go:163,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2010
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: output in %s
		    -- arg 1: <string>
		  - output in run_074153.168_n4_workload_check_tpcc:
		  - error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:406
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2019
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2000
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1819517-1584688629-01-n4cpu4:4 -- ./workload check tpcc --warehouses 500 --expensive-checks {pgurl:1} returned:
		    stderr:
		    I200320 07:41:54.802818 1 workload/tpcc/tpcc.go:372  check 3.3.2.1 took 68.0927ms
		    I200320 07:42:06.647382 1 workload/tpcc/tpcc.go:372  check 3.3.2.2 took 11.844515639s
		    I200320 07:42:08.828918 1 workload/tpcc/tpcc.go:372  check 3.3.2.3 took 2.181486691s
		    I200320 07:43:08.591889 1 workload/tpcc/tpcc.go:372  check 3.3.2.4 took 59.762923849s
		    I200320 07:45:34.879397 1 workload/tpcc/tpcc.go:372  check 3.3.2.5 took 2m26.287442343s
		    I200320 07:50:06.000105 1 workload/tpcc/tpcc.go:372  check 3.3.2.6 took 4m31.120607137s
		    Error: check failed: 3.3.2.6: pq: communication error: rpc error: code = Canceled desc = context canceled
		    Error:  exit status 1
		    
		    stdout::
		  - exit status 1

```

<details><summary>More</summary><p>


Artifacts: [/alterpk-tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1819517&tab=artifacts#/alterpk-tpcc)
Related:
- #46353 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003200044_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003200044_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46317 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003181957_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003181957_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46136 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003112118_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003112118_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aalterpk-tpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).alterpk-tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1821791&tab=buildLog) on [master@322f83364bc0807c3147af0631cb7317c584ce43](https://github.com/cockroachdb/cockroach/commits/322f83364bc0807c3147af0631cb7317c584ce43):

```
	cluster.go:1935,alterpk.go:163,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2010
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: output in %s
		    -- arg 1: <string>
		  - output in run_073103.139_n4_workload_check_tpcc:
		  - error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:406
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2019
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2000
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1821791-1584774229-06-n4cpu4:4 -- ./workload check tpcc --warehouses 500 --expensive-checks {pgurl:1} returned:
		    stderr:
		    I200321 07:31:04.800590 1 workload/tpcc/tpcc.go:372  check 3.3.2.1 took 81.949461ms
		    I200321 07:31:13.169282 1 workload/tpcc/tpcc.go:372  check 3.3.2.2 took 8.368647632s
		    I200321 07:31:14.673128 1 workload/tpcc/tpcc.go:372  check 3.3.2.3 took 1.503771706s
		    I200321 07:32:19.538194 1 workload/tpcc/tpcc.go:372  check 3.3.2.4 took 1m4.865018024s
		    I200321 07:34:41.273991 1 workload/tpcc/tpcc.go:372  check 3.3.2.5 took 2m21.735694083s
		    I200321 07:39:26.693407 1 workload/tpcc/tpcc.go:372  check 3.3.2.6 took 4m45.419344785s
		    Error: check failed: 3.3.2.6: pq: communication error: rpc error: code = Canceled desc = context canceled
		    Error:  exit status 1
		    
		    stdout::
		  - exit status 1

```

<details><summary>More</summary><p>


Artifacts: [/alterpk-tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1821791&tab=artifacts#/alterpk-tpcc)
Related:
- #46317 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003181957_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003181957_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46136 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003112118_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003112118_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aalterpk-tpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).alterpk-tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1826302&tab=buildLog) on [master@6c3c5d4a9a0b837538a558fa22dcd4ff712fd8f5](https://github.com/cockroachdb/cockroach/commits/6c3c5d4a9a0b837538a558fa22dcd4ff712fd8f5):

```
	cluster.go:1935,alterpk.go:163,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2010
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: output in %s
		    -- arg 1: <string>
		  - output in run_075500.038_n4_workload_check_tpcc:
		  - error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:406
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2019
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2000
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1933
		    main.registerAlterPK.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/alterpk.go:163
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1826302-1585121307-05-n4cpu4:4 -- ./workload check tpcc --warehouses 500 --expensive-checks {pgurl:1} returned:
		    stderr:
		    I200325 07:55:01.649465 1 workload/tpcc/tpcc.go:372  check 3.3.2.1 took 39.28485ms
		    I200325 07:55:07.798830 1 workload/tpcc/tpcc.go:372  check 3.3.2.2 took 6.149313448s
		    I200325 07:55:10.097753 1 workload/tpcc/tpcc.go:372  check 3.3.2.3 took 2.298875719s
		    I200325 07:57:21.497403 1 workload/tpcc/tpcc.go:372  check 3.3.2.4 took 2m11.399605056s
		    I200325 07:59:39.041365 1 workload/tpcc/tpcc.go:372  check 3.3.2.5 took 2m17.543862907s
		    I200325 08:12:04.625868 1 workload/tpcc/tpcc.go:372  check 3.3.2.6 took 12m25.584429082s
		    Error: check failed: 3.3.2.6: pq: communication error: rpc error: code = Canceled desc = context canceled
		    Error:  exit status 1
		    
		    stdout::
		  - exit status 1

```

<details><summary>More</summary><p>


Artifacts: [/alterpk-tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1826302&tab=artifacts#/alterpk-tpcc)
Related:
- #46457 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003240059_v20.1.0-beta.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003240059_v20.1.0-beta.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46317 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003181957_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003181957_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46136 roachtest: alterpk-tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003112118_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003112118_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aalterpk-tpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/alterpk.go
+++ b/pkg/cmd/roachtest/alterpk.go
@@ -89,7 +89,8 @@ func registerAlterPK(r *testRegistry) {
 
 	// runAlterPKTPCC runs a primary key change while the TPCC workload runs.
 	runAlterPKTPCC := func(ctx context.Context, t *test, c *cluster) {
-		const warehouses = 500
+		// TODO(yuzefovich): increase this back to 500 once #47205 is resolved.
+		const warehouses = 100
 		const duration = 10 * time.Minute
 
 		roachNodes, loadNode := setupTest(ctx, t, c)

